### PR TITLE
NIAD-858: logging

### PR DESCRIPTION
**Pull request by bartek-sarul**
## Description

Change conversation id with correlation id.
Add logback.

## Jira Ticket

https://gpitbjss.atlassian.net/browse/NIAD-858

## Checklist

These are items (excluding GitHub Checks) which should be confirmed before a branch is ready to merge.

- [ ] Acceptance Criteria met
- [ ] Commit messages are meaningful
- [ ] Manually tested
- [ ] Self-reviewed your code
- [ ] Code reviewed from two other developers
- [ ] If your pull request depends on any other, please link them in the description
- [ ] main branch is passing/green on CI
- [ ] Add/update any relevant documentation

### CODE REVIEW

**Review by andrewwa-kainos — APPROVED** [review]
LGTM

File: src/main/resources/application.yml [hunk 1]
```diff
@@ -25,37 +23,118 @@ spring:
 
 labresults:
   amqp:
-    brokers: ${LAB_RESULTS_AMQP_BROKERS}
-    meshOutboundQueueName: ${LAB_RESULTS_MESH_OUTBOUND_QUEUE_NAME}
-    meshInboundQueueName: ${LAB_RESULTS_MESH_INBOUND_QUEUE_NAME}
-    gpOutboundQueueName: ${LAB_RESULTS_GP_OUTBOUND_QUEUE_NAME}
+    brokers: ${LAB_RESULTS_AMQP_BROKERS:amqp://localhost:5672}
+    meshOutboundQueueName: ${LAB_RESULTS_MESH_OUTBOUND_QUEUE_NAME:lab-results_mesh_outbound}
```
Comment (saileshp-kainos) [inline src/main/resources/application.yml:27]:
```suggestion
    meshOutboundQueueName: ${LAB_RESULTS_MESH_OUTBOUND_QUEUE_NAME:lab_results_mesh_outbound}
```

File: src/main/resources/application.yml [hunk 2]
```diff
@@ -25,37 +23,118 @@ spring:
...
+    meshInboundQueueName: ${LAB_RESULTS_MESH_INBOUND_QUEUE_NAME:lab-results_mesh_inbound}
```
Comment (saileshp-kainos) [inline src/main/resources/application.yml:28]:
```suggestion
    meshInboundQueueName: ${LAB_RESULTS_MESH_INBOUND_QUEUE_NAME:lab_results_mesh_inbound}
```

File: src/main/resources/application.yml [hunk 3]
```diff
@@ -25,37 +23,118 @@ spring:
...
+    gpOutboundQueueName: ${LAB_RESULTS_GP_OUTBOUND_QUEUE_NAME:lab-results_gp_outbound}
```
Comment (saileshp-kainos) [inline src/main/resources/application.yml:29]:
```suggestion
    gpOutboundQueueName: ${LAB_RESULTS_GP_OUTBOUND_QUEUE_NAME:lab_results_gp_outbound}
```

**Review by saileshp-kainos — CHANGES_REQUESTED** [review]
The queue names here are inconsistent with the ones set in docker-compose and in other configurations where an underscore is used between lab & result

**Review by andrewwa-kainos — APPROVED** [review]

**Review by saileshp-kainos — APPROVED** [review]
LGTM

**Pull request merged**